integration/common

```diff
@@ -34,164 +34,45 @@ waitForResponse () {
+  popd 
 
-  yarn workspace @chainlink/integration-scripts fund-address > $SRCROOT/integration/logs/fund_address.log
-}
-
-explorer_url="http://127.0.0.1:8080"
-
-launch_explorer() {
-  printf -- "\033[34m%-50s\033[0m" "Checking for explorer"
-  if curl -s $explorer_url >/dev/null; then
-    printf -- "[\033[32mrunning\033[0m]\n"
-    return
-  fi
-  printf -- "[\033[31mstopped\033[0m]\n"
-  title "Starting explorer..."
-
-  yarn workspace @chainlink/explorer run build &>$SRCROOT/integration/logs/explorer-yarn.log
-  yarn workspace @chainlink/explorer run prod &>$SRCROOT/integration/logs/explorer.log & waitForResponse $explorer_url
-  title "Explorer is running."
-}
-
-add_clnode_to_explorer() {
-  title "Adding CL Node to Explorer..."
-
-  yarn workspace @chainlink/explorer admin:seed test-admin test-admin-password
-  clnode_name="test-$(date +%s)"
-  response=`EXPLORER_BASE_URL=$explorer_url EXPLORER_ADMIN_USERNAME=test-admin EXPLORER_ADMIN_PASSWORD=test-admin-password yarn workspace @chainlink/explorer run admin:clnodes:add $clnode_name`
-  export EXPLORER_ACCESS_KEY=$(echo $response | grep -o "AccessKey [a-zA-Z0-9]*" | awk '{print $2}')
-  export EXPLORER_SECRET=$(echo $response | grep -o "Secret [a-zA-Z0-9]*" | awk '{print $2}')
-  export EXPLORER_URL="ws://localhost:8080"
-  title "Added CL Node $clnode_name to Explorer."
-}
-
-setup_scripts() {
-  yarn --no-progress install &>$SRCROOT/integration/logs/yarn.log
-  yarn workspace chainlinkv0.5 build
-  yarn workspace chainlink build
-  yarn workspace @chainlink/integration-scripts setup
+  yarn workspace @chainlink/integration-scripts fund-address
 }
 
 deploy_contracts() {
   title "Migrating core contracts..."
   pushd integration >/dev/null
 
   # run migrations
-  yarn workspace @chainlink/integration-scripts deploy-contracts | tee $SRCROOT/integration/logs/deploy.log
-  export LINK_TOKEN_ADDRESS=`cat $SRCROOT/integration/logs/deploy.log | grep 'Deployed LinkToken at:' | awk '{print$4}'`
-  export ORACLE_CONTRACT_ADDRESS=`cat $SRCROOT/integration/logs/deploy.log | grep 'Deployed Oracle at:' | awk '{print$4}'`
-  export ETH_LOG_ADDRESS=`cat $SRCROOT/integration/logs/deploy.log | grep 'Deployed EthLog at:' | awk '{print$4}'`
-  export RUN_LOG_ADDRESS=`cat $SRCROOT/integration/logs/deploy.log | grep 'Deployed RunLog at:' | awk '{print$4}'`
+  yarn workspace @chainlink/integration-scripts deploy-contracts | tee /chainlink/integration/logs/deploy.log
+  export LINK_TOKEN_ADDRESS=`cat /chainlink/integration/logs/deploy.log | grep 'Deployed LinkToken at:' | awk '{print$4}'`
+  export ORACLE_CONTRACT_ADDRESS=`cat /chainlink/integration/logs/deploy.log | grep 'Deployed Oracle at:' | awk '{print$4}'`
+  export ETH_LOG_ADDRESS=`cat /chainlink/integration/logs/deploy.log | grep 'Deployed EthLog at:' | awk '{print$4}'`
+  export RUN_LOG_ADDRESS=`cat /chainlink/integration/logs/deploy.log | grep 'Deployed RunLog at:' | awk '{print$4}'`
```

Comment:
why is the root /chainlink chosen? '/' is usually for system files

Reply:
I chose this path since it would be short, not sure what the proper path would be in this case since its running in a container